src/applications/caregivers/components/PreSubmitInfo/SignatureInput.jsx

```diff
@@ -1,6 +1,6 @@
-import React, { useEffect, useState } from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
-import TextInput from '@department-of-veterans-affairs/component-library/TextInput';
+import { VaTextInput } from 'web-components/react-bindings';
```

Comment:
I think that the Design System team recommends we use the full import

```js
import { VaTextInput } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
```

Reply:
Isn't it better to use the alias though? 90% of the imports for `react-bindings` in vets-website use the alias

Reply:
Even the rules for migrating components with `eslint` use `web-components/react-bindings`. https://github.com/department-of-veterans-affairs/vets-website/blob/master/script/eslint-plugin-va/rules/prefer-web-component-library.js#L201

Reply:
I'm going off of the [recommendation Brooks gave me in an older PR](https://github.com/department-of-veterans-affairs/vets-website/pull/20709#discussion_r840021556)

Reply:
Ah ok, thanks for posting that. Looks like I’ll be updating some imports tomorrow